### ISSUE TITLE
Missing focus outlines

### ISSUE THREAD

**Issue by oliverjam** [fac22/week2-ABES#25]
https://github.com/fac22/week2-ABES/blob/2b46c229a651c0ef3a2d5b642205203226c1c97a/public/styles.css#L170-L171

If you disable focus outlines for interactive elements it's important to ensure you replace them with something accessible. Unfortunately this rule is broken:

https://github.com/fac22/week2-ABES/blob/2b46c229a651c0ef3a2d5b642205203226c1c97a/public/styles.css#L180-L183

since the `outline` shorthand property needs a width and style (the same as `border`). So no outline shows up when you focus this input.
